modules/puppet_proxy_customrun/puppet_proxy_customrun_plugin.rb

```diff
@@ -0,0 +1,10 @@
+module Proxy::PuppetCustomrun
+  class Plugin < Proxy::Provider
+    default_settings :customrun_cmd => '/bin/false', :customrun_args => "-ay -f -s"
```

Comment:
Perhaps just leave these out and require them to be set?  I don't think they're useful defaults, just a vague example from the config file.